src/index.html

```diff
@@ -10,7 +11,49 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search"
+      action="#"
+      method="get"
```

Comment:
it's better to use POST method instead of GET

Reply:
You totally right, I don't know why I used here GET (it's default)... As I know GET will show data while transfer it in page link, and POST will encrypt it. Probably I think that this data isn't sensitif...